src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const discount = 20;
+  const bigDiscount = 50;
```

Comment:
The variable name 'bigDiscount' does not clearly describe the purpose or the amount of the discount. Consider renaming it to 'largeDiscount' or 'discountFor7days'. 🤔